gapic/api/artman_lro.yaml

```diff
@@ -0,0 +1,25 @@
+common:
+  api_name: google-longrunning-operations
```

Comment:
Fair enough. That also shortens it without resorting to an arbitrary abbreviation (e.g. `lro`).